commands/build.go

```diff
@@ -91,9 +94,39 @@ func preRunBuild(cmd *cobra.Command, args []string) error {
 		buildArgMap = mapped
 	}
 
+	buildLabelMap, err = parseBuildLabel(buildLabels)
```

Comment:
Can you add an example to the `Example` field of the `buildCmd`. I'd use `--build-label "mykey=myval"` (or whatever gets chosen for `--build-label`.

Reply:
Sure I will add it.